pkg/event/target/webhook.go

```diff
@@ -209,14 +220,20 @@ func NewWebhookTarget(id string, args WebhookArgs, doneCh <-chan struct{}, logge
 	var store Store
 
 	target := &WebhookTarget{
-		id:   event.TargetID{ID: id, Name: "webhook"},
-		args: args,
-		httpClient: &http.Client{
-			Transport: transport,
-		},
+		id:         event.TargetID{ID: id, Name: "webhook"},
+		args:       args,
 		loggerOnce: loggerOnce,
 	}
 
+	if target.args.ClientCert != "" && target.args.ClientKey != "" {
+		c, err := certs.New(target.args.ClientCert, target.args.ClientKey, config.LoadX509KeyPair)
```

Comment:
just use tls.LooadX509KeyPair - which is enough here, supporting what config.LoadX509KeyPair is perhaps an overkill?

Reply:
I added because it had the `MINIO_CERT_PASSWD` support. But, I feel that would be an overkill too. Removed it.